src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -66,18 +67,22 @@ of mol photons per m^2 per second (`APAR`).
 
 This applies the two-stream radiative transfer solution which takes into account
 the impacts of scattering within the canopy. The function takes in all 
-paramters from the parameter struct of a TwoStreamModel, along with the PAR, 
-LAI, extinction coefficient K, and solar zenith anlgle.
+paramters from the parameter struct of a TwoStreamModel which it extracts from
+the canopy model, along with the incident PAR, LAI, extinction coefficient K, 
+and solar zenith angle.
 """
 function plant_absorbed_ppfd(
-    RT::TwoStreamModel,
+    canopy::CanopyModel{FT, TwoStreamModel{FT}},
     PAR::FT,
     LAI::FT,
     K::FT,
     θs::FT,
```

Comment:
A couple of other options for these two methods:
- we can dispatch off of parametric types as well. so, I think that if we do pass canopy as an argument, we dont additionally need to pass the RT model as an argument, because we can dispatch off of the parametric type in canopy. A little care needs to be taken to make sure this works properly though. maybe something like...```canopy::CanopyModel{FT, <: TwoStreamModel}```
- however, I think we may also consider passing in albedo (a float) as the last argument and dispatching off of the radiative transfer model, and *not* passing in canopy (unless we think we need the whole model for another reason). The method for plant_absorbed_ppfd() for the BeerLambertModel can then be
```
plant_absorbed_ppfd(
    RT::BeerLambertModel,
    PAR::FT,
    LAI::FT,
    K::FT,
    θs::FT
     _)
```
i.e. it doesnt use the last argument (albedo), even though we always pass it. And, prior to this function being called (in Canopy.jl), we compute albedo to pass in. The reason why I suggest this is because albedo will eventually vary in space, and be stored in the aux state, and not just be a parameter that is set once. 

I am open to leaving this PR as is, though, and addressing this in a future PR when we get there :)